pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -34,7 +33,7 @@
 
 buildFun:
 
-with stdenv.lib;
+with stdenv_32bit.lib;
```

Comment:
Again, I'd like to avoid requiring multilib just for NaCl, if it only works with a 32bit toolchain then we really should separate it.